fluent-plugin-logtail.gemspec

```
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'fluent-plugin-logtail'
-  s.version     = '0.1.1'
+  s.version     = '0.1.2'
```

It seems we already have `0.2.0` released - https://rubygems.org/gems/fluent-plugin-logtail/versions/0.2.0

Looks like the version wasn't bumped in 778fc642d6175745c80bb9f1fbe2d6b1c1f4dc7f

```suggestion
  s.version     = '0.2.1'
```